plugins/ccxtExchange_test.go

```diff
@@ -587,28 +581,20 @@ func TestGetOrderConstraints_Ccxt_Precision(t *testing.T) {
 		},
 	}
 
-	for _, kase := range testCases {
-		t.Run(kase.exchangeName, func(t *testing.T) {
-			testCcxtExchange, e := makeCcxtExchange(
-				kase.exchangeName,
-				nil,
-				[]api.ExchangeAPIKey{emptyAPIKey},
-				[]api.ExchangeParam{emptyParams},
-				[]api.ExchangeHeader{},
-				false,
-				getEsParamFactory(kase.exchangeName),
-			)
-			if !assert.NoError(t, e) {
-				return
-			}
+	for _, tc := range testCases {
+		t.Run(tc.exchangeName, func(t *testing.T) {
 
-			result := testCcxtExchange.GetOrderConstraints(kase.pair)
-			if !assert.Equal(t, kase.wantPricePrecision, result.PricePrecision) {
-				return
-			}
-			if !assert.Equal(t, kase.wantVolPrecision, result.VolumePrecision) {
-				return
-			}
+			epf := getEsParamFactory(tc.exchangeName)
+
+			ccxtExchange, e := makeCcxtExchange(tc.exchangeName, nil, []api.ExchangeAPIKey{emptyAPIKey},
+				[]api.ExchangeParam{emptyParams}, []api.ExchangeHeader{}, false, epf)
+
+			require.NoError(t, e)
```

Comment:
Good idea to use require, it’s exactly the behavior we want. We should migrate to using require in other places too long-term. I’ll do that at some point